errata/common.yaml

```diff
@@ -10,16 +10,7 @@ GAP/SEC/SEM/BV-66-C: TSE18349
 GAP/SEC/SEM/BV-67-C: TSE18349
 GAP/DM/NAD/BV-02-C: TSE19171
 GAP/DM/LEP/BV-06-C: TSE19171
-GAP/PADV/PASM/BV-01-C: https://github:com/intel/auto-pts/issues/740
-GAP/PADV/PAM/BV-01-C: https://github:com/intel/auto-pts/issues/740
-GAP/PADV/PASE/BV-01-C: https://github:com/intel/auto-pts/issues/740
-GAP/PADV/PASE/BV-02-C: https://github:com/intel/auto-pts/issues/740
-GAP/PADV/PASE/BV-03-C: https://github:com/intel/auto-pts/issues/740
-GAP/PADV/PASE/BV-04-C: https://github:com/intel/auto-pts/issues/740
-GAP/PADV/PASE/BV-05-C: https://github:com/intel/auto-pts/issues/740
-GAP/PADV/PASE/BV-06-C: https://github:com/intel/auto-pts/issues/740
-GAP/PADV/PAST/BV-01-C: https://github:com/intel/auto-pts/issues/740
-GAP/PADV/PAST/BV-02-C: https://github:com/intel/auto-pts/issues/740
+GAP/PADV/PASE/BV-01-C: https://github.com/apache/mynewt-nimble/issues/1512
```

Comment:
it this is not a PTS or TSE issue lets just not rack it here (so simply remove those entries)